WordPress/src/main/res/values/strings.xml

```diff
@@ -1480,6 +1480,7 @@
     <string name="error_refresh_unauthorized_posts">You don\'t have permission to view or edit posts</string>
     <string name="error_fetch_my_profile">Couldn\'t retrieve your profile</string>
     <string name="error_fetch_account_settings">Couldn\'t retrieve your account settings</string>
+    <string name="error_disabled_apis">Could not fetch settings: Some APIs are disabled for this account + client ID combination.</string>
```

Comment:
> Some APIs are disabled for this account + client ID combination.

When I saw this, I thought there was something wrong with my actual WordPress user. What do you think about this? 

```
Could not fetch settings: Some APIs are unavailable for this OAuth app ID and client ID combination.
```

The word "disable" connotes that it can be enabled somewhere but I believe it is impossible or we can't ever allow it, right?

Reply:
Hmm I see your point 🤔 but the OAuth app ID and client ID are the same thing. I think OAuth app ID is better for the message than client ID though since that's what we called the param in `gradle.properties`. 👍

What if we emphasized the 'OAuth app ID' part?

```
Could not fetch settings: Some APIs are unavailable for this OAuth app ID + account combination.
```

> The word "disable" connotes that it can be enabled somewhere but I believe it is impossible or we can't ever allow it, right?

Basically, yes. I read it more as "works by default but not in this case", but "unavailable" works just as well for me and has less of a "default" connotation, which is maybe better. 👍 Will use it in the final message once we settle on one.

Reply:
Oh, oops. I meant `secret` instead of `client ID` there. 😅 But yep, your suggested message is great for me! 

```
Could not fetch settings: Some APIs are unavailable for this OAuth app ID + account combination.
```